cmd/conduit/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"fmt"
+	"github.com/algorand/indexer/conduit"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+)
+
+import (
+	// We need to import these so that the package wide init() function gets called
+	_ "github.com/algorand/indexer/exporters/all"
+	_ "github.com/algorand/indexer/importers/all"
+	_ "github.com/algorand/indexer/processors/all"
+)
+
+var (
+	logger *log.Logger
+)
+
+// init() function for main package
+func init() {
+
+	// Setup logger
+	logger = log.New()
+
+	formatter := conduit.PluginLogFormatter{
+		Formatter: &log.JSONFormatter{
+			DisableHTMLEscape: true,
+		},
+		Type: "Conduit",
+		Name: "main",
+	}
+
+	logger.SetFormatter(&formatter)
+	logger.SetOutput(os.Stdout)
+}
+
+// runConduitCmdWithConfig run the main logic with a supplied conduit config
+func runConduitCmdWithConfig(cfg *conduit.Config) error {
```

Comment:
having viper/pflag stuff in the run function might make it more difficult to use this in daemon later on. I think this can just accept a single `dataDir string` parameter.

Reply:
Should we manually check `CONDUIT_DATA_DIR` since it's the only special case?